java/client/src/main/java/glide/connectors/handlers/CallbackDispatcher.java

```diff
@@ -102,15 +102,19 @@ public void completeRequest(Response response) throws MessageHandler.MessageCall
                     case Unspecified:
                         // Unspecified error on Redis service-side
                         future.completeExceptionally(new RequestException(msg));
+                        break;
```

Comment:
🤦 
Do we need to update tests after this fix? If no - some tests are missing.

Reply:
Integration and unit tests pass, which mean we have a coverage hole.